rca/forms/models.py

```diff
@@ -54,7 +67,37 @@ class FormPage(WagtailCaptchaEmailForm, BasePage):
                     ]
                 ),
                 FieldPanel("subject"),
+                FieldPanel("send_user_notification"),
+                FieldPanel("email_body_copy"),
             ],
             "Email",
         ),
     ]
+
+    def process_form_submission(self, form):
+        submission = super().process_form_submission(form)
+        if self.send_user_notification:
+            self.send_user_mail(form)
+        return submission
+
+    def send_user_mail(self, form):
+        """Sends the email notification to the user who submitted the form.
+        Depends on there being a form field labelled 'Email'"""
+        address = form.cleaned_data.get('email')
+        if address:
+            send_mail(self.subject, self.render_email(form), [address], self.from_address,)
+
+    def render_email(self, form):
+        responses = super().render_email(form)
+
+        content = [
+            "Hi,",
+            self.email_body_copy,
+            "See below the responses you submitted:",
```

Comment:
```suggestion
            "See below for the responses you submitted:",
```
Disclaimer, I'm not hot on grammar